frontend/templates/views/partials/breadcrumbs.j2

```diff
@@ -70,10 +70,10 @@ and the current page title #}
 
   {% set courses = g.doc('/content/amp-dev/documentation/courses/index.html', locale=doc.locale) %}
   {% if not doc.collection.bucket %}
-    <a class="ap-m-breadcrumbs-crumb" href="{{ courses.url.path }}">{{ courses.titles('navigation') }}</a>
+    <a class="ap-m-breadcrumbs-crumb" href="{{ courses.url.path }}">{{ _(courses.titles('navigation')) }}</a>
```

Comment:
Similar as with the working groups: `courses.titles('navigation')` will already be localized as we are fetching the localized document in the `{% set %}` statement above. For it to work we would actually need to create a translated courses document.